gumby/experiments/TriblerDispersyClient.py

```diff
@@ -59,7 +59,7 @@ def _do_start():
 
             logging.error("Tribler Session started")
             self.annotate("Tribler Session started")
-
+    
```

Comment:
Remove this and make sure you remove all trailing white spaces from all files you create in this PR.